packages/zowe-explorer/src/trees/job/JobActions.ts

```diff
@@ -457,9 +454,15 @@ export class JobActions {
             }
         }
 
+        for (const session of sessionNodes) {
+            session.dirty = true;
+            await session.getChildren();
+            jobsProvider.refreshElement(session);
+        }
+
         if (failedJobs.length > 0) {
             // Display any errors from the API
-            await Gui.warningMessage(
+            Gui.warningMessage(
```

Comment:
Should we leave a comment in the code as to way we are not awaiting these methods?
And maybe disable the eslint rule for that line 😋

Reply:
Solved in f8c068de